src/components/Header.tsx

```diff
@@ -0,0 +1,41 @@
+import Link from 'next/link';
+import { useState } from 'react';
+import HeaderLobby from './HeaderLobby';
+import HeaderUtilities from './HeaderUtilities';
+import CritSVG from '../../public/images/critLogo.svg';
+
+const Header = () => {
+  const [isLoggedIn, setIsLoggedIn] = useState(true);
+
+  return (
+    <header className="header">
+      <div className="header__left">
+        <Link href={'/home'} passHref>
+          <div className="header__image">
+            <CritSVG />
+            <h2>Crit</h2>
+          </div>
+        </Link>
+        <div className="header__message-of-the-day">
+          <p>Your pretty awesome</p>
+        </div>
```

Comment:
Would you please add a TODO here so we can have some API endpoint or some way to randomize messages in the future?